### PR TITLE
Acclamoji

### PR DESCRIPTION
**Pull request by kcboschert**
Adds emoji support for cards on the card wall.

Solves https://github.com/matthewpatterson/acclamation/issues/1

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "jquery.finger": "https://github.com/matthewpatterson/jquery.finger.git#master",
     "jquery.ui": "1.11.1",
     "pure": "0.5.0",
-    "tinysort": "*"
+    "tinysort": "*",
+    "jquery-textcomplete": "~0.3.6",
+    "emoji-parser": "~0.1.1"
```
Comment (atmattpatt) [inline bower.json]: Could you lock these to the specific version (kill the `~`)

File: Gruntfile.js [hunk 2]
```diff
@@ -144,6 +157,13 @@ module.exports = function(grunt) {
       },
       stop_redis: {
         command: 'pgrep redis-server | xargs kill'
+      },
+
+      emojify: {
+        command: 'rsync -az bower_components/emoji-parser/emoji public/images && echo "var base_emojis = [" > public/javascripts/base_emoji.js && ls -1 public/images/emoji | grep ".png" | sed -e \'s/\\..*$//\' | xargs -I{} echo \\\"{}\\\" | tr \"\\n\" "," | sed "s/,$//" >> public/javascripts/base_emoji.js && echo "];" >> public/javascripts/base_emoji.js',
```
Comment (atmattpatt) [inline Gruntfile.js:163]: :linux_prize:

File: src/views/client.ejs [hunk 3]
```diff
@@ -8,6 +8,8 @@
     <link rel="stylesheet" type="text/css" href="/stylesheets/acclamation.css"/>
     <script type="text/javascript" src="/socket.io/socket.io.js"></script>
     <script type="text/javascript" src="/javascripts/client.js"></script>
+    <script type="text/javascript" src="/javascripts/base_emoji.js"></script>
+    <script type="text/javascript" src="/javascripts/emoji.js"></script>
```
Comment (atmattpatt) [inline src/views/client.ejs]: Shouldn't `emoji.js` be getting included as part of `client.js`?

**Comment by atmattpatt** [comment]
Aside from the couple of small things I noted, looks good!  :frog: :dango: :moneybag:

File: src/views/client.ejs (hunk 3 above)
Comment (kcboschert) [inline src/views/client.ejs]: Ah, you're right. I forgot to remove this before pushing.

File: Gruntfile.js (hunk 2 above)
Comment (kcboschert) [inline Gruntfile.js:163]: Only managed some of this on my own. Had a little help with the xargs bit. Probably a simpler way to do it, but works for now.

Comment (atmattpatt) [inline Gruntfile.js:163]: Yeah I'm thinking there's probably a way to use Node's fs module to list the files, and then just `JSON.stringify` the thing.  But I'm also fine with this for now.

File: bower.json (hunk 1 above)
Comment (kcboschert) [inline bower.json]: Fixed 5ab12beba83dde8162444be328deff1e9a138457

**Pull request merged**